webapp/start.sh

```diff
@@ -76,14 +76,19 @@ for setting in $(compgen -A variable KCCONF_WEBAPPPLUGIN_); do
 	php_cfg_gen /etc/kopano/webapp/config-"$identifier".php "${setting3}" "${!setting}"
 done
 
+# disable tls internally
+sed -i '/#tls /c\tls = no' /etc/kopano/kwebd.cfg
+
+# load kweb cfg into env for kopano-kwebd
+sed -i s/\ *=\ */=/g /etc/kopano/kwebd.cfg
+# shellcheck disable=SC2046
+export $(grep -v '^#' /etc/kopano/kwebd.cfg | xargs -d '\n')
+
 echo "Ensure config ownership"
 chown -R www-data:www-data /run/sessions /tmp/webapp
 
-echo "Starting Apache"
-rm -f /run/apache2/apache2.pid
-set +u
-# shellcheck disable=SC1091
-source /etc/apache2/envvars
 # cleaning up env variables
 unset "${!KCCONF_@}"
-exec /usr/sbin/apache2 -DFOREGROUND
+echo "Starting php-fpm"
+/usr/sbin/php-fpm7.0 -F &
```

Comment:
It would look nicer in the process tree (like ps), if you would call long running processes without their paths.

Reply:
+1